docs/sphinx/source/whatsnew/v0.9.2.rst

```diff
@@ -11,6 +11,9 @@ Enhancements
 * Add :py:func:`pvlib.tracking.calc_surface_orientation` for calculating
   single-axis tracker ``surface_tilt`` and ``surface_azimuth`` from
   rotation angles. (:issue:`1471`, :pull:`1480`)
+* Add :py:func:`pvlib.iotools.read_solaranywhere` and
+  :py:func:`pvlib.iotools.get_solaranywhere` for reading and retrieving
+  SolarAnywhere solar irradiance data. (:pull:`1497`, :discussion:`1310`)
```

Comment:
This will need a new [extlinks](https://www.sphinx-doc.org/en/master/usage/extensions/extlinks.html) entry for `discussion` here: 

https://github.com/pvlib/pvlib-python/blob/3f397ed44075b01f4ba535750356859195a51b2d/docs/sphinx/source/conf.py#L295-L301

Reply:
Perhaps it is more appropriate to convert the discussion to an issue?